docs/rules/no-unsupported-features/es-builtins.md

```diff
@@ -24,9 +24,9 @@ This rule gets the supported Node.js version range from the following, falling b
 1. Rule configuration `version`
 2. ESLint [shared setting](http://eslint.org/docs/user-guide/configuring.html#adding-shared-settings) `node.version`
 3. `package.json` [`engines`] field
-4. `>=8.0.0`
+4. `>=16.0.0`
 
-The default version is `8.0.0` because it's the minimum version the community is maintaining (see also [Node.js Release Working Group](https://github.com/nodejs/Release#readme)).
+If you omit the [engines] field, this rule chooses `>=16.0.0` as the configured Node.js version since `16` is the active lts (see also [Node.js Release Working Group](https://github.com/nodejs/Release#readme)).
```

Comment:
Same here as in my other comment. Could we maybe move the documentation of the default to a single place and refer to it in all the other places rather than duplicate the documentation of the default?

Duplicates docs always risks getting out of sync

Reply:
it's also not good to maintain - to update it, I had to find/replace it in the whole repo.

👍 sounds good! will do.